src/App.js

```diff
@@ -1,19 +1,55 @@
 import React from 'react';
-import './App.css';
 
+import './App.css';
+import todos from './api/todos';
 import users from './api/users';
+import TodoList from './components/TodoList';
+import NewTodo from './components/NewTodo';
+
+function userListofTodo(ListOfTodo) {
+  const userInfo = userId => users.find(user => user.id === userId);
+
+  return ListOfTodo.map(list => (
+    {
+      ...list,
+      user: userInfo(list.userId),
+    }));
```

Comment:
странные переносы, переноси красиво и логично
```suggestion
  return ListOfTodo.map(list => ({
      ...list,
      user: userInfo(list.userId),
    }));
```